### PR TITLE
battle cards

### PR DESCRIPTION
**Pull request by kpathakota**
Battle cards.

Any other sections needed for each one? Hopefully good enough for a first pass.

### CODE REVIEW

**Comment by codefriar** [comment]
@kpathakota I, for one, would love to see this lovingly presented as a table.

Let's see if my markdown table skills are up to snuff.

|Software Name| Hierarchy | open source | foo | bar |
| Dendron | [x] | [x] | [x] | [x] |
| Obsidian | [ ] | [ ] | [ ] | [ ] |

etc.

**Comment by kpathakota** [comment]
@codefriar - yes, can do. Like the idea.

**Comment by kpathakota** [comment]
@kevinslin - great point on the refs - much cleaner now! Should be good to merge from my end if you wouldn't mind doing a cursory check. 

 @codefriar - hope you like the table :)

File: vault/dendron.alternatives.md [hunk 1]
```diff
@@ -1,18 +1,24 @@
...
+All the tools below are excellent for their own use-cases. This comparison rests purely where Dendron is different.
 
-![[dendron.alternatives.*]]
\ No newline at end of file
+| Software Name | Comparison w/ Dendron                        | Flexible Hierarchy | Open source | Local-first | Fast and performant |
+| ------------- | -------------------------------------------- | ------------------ | ----------- | ----------- | ------------------- |
+| Dendron       | How?                                         | ✅                 | ✅          | ✅          | ✅                  |
+| Notion        | [[dendron.alternatives.notion-vs-dendron]]   | ✅                 | ❌          | ❌          | ✅                  |
+| Obsidian      | [[dendron.alternatives.obsidian-vs-dendron]] | ❌                 | ❌          | ✅          | ✅                  |
+| Roam          | [[dendron.alternatives.roam-vs-dendron]]     | ❌                 | ❌          | ❌          | ❌                  |
```
Comment (kevinslin) [inline vault/dendron.alternatives.md:24]: this is a bit unfair to roam. something I would add to feature list is perhaps daily journals and bidirectional links

Comment (kevinslin) [reply vault/dendron.alternatives.md:24]: also zoomable outlining (not sure what the term is), which only roam supports from this list at this point

Comment (kpathakota) [reply vault/dendron.alternatives.md]: Good call out - Notion can also be used for outlining/page within a page so I gave it a green checkmark as well.

Comment (ChristinWhite) [reply vault/dendron.alternatives.md]: @kevinslin Unless I misunderstanding what you mean by zoomable outlining — I haven't used Roam — at this point Obsidian has support for it through the [Outliner](https://github.com/vslinko/obsidian-outliner) and [Zoom](https://github.com/vslinko/obsidian-zoom) plugins.

**Comment by ChristinWhite** [comment]
I'd love to see FOAM and some of the other semi-mature VSCode systems in the comparison.

**Comment by kevinslin** [comment]
Didn't realize the obsidian plugin. We should add a `Check with asterix` to
indicate its supported via plugins.
And also agree about adding Foam to the list

**Pull request merged**